fdbserver/ptxn/TLogPeekCursor.actor.cpp

```diff
@@ -114,7 +114,7 @@ ACTOR Future<bool> peekRemote(PeekRemoteContext peekRemoteContext) {
 	        .pTLogInterfaces[deterministicRandom()->randomInt(0, peekRemoteContext.pTLogInterfaces.size())];
 
 	request.debugID = peekRemoteContext.debugID;
-	request.beginVersion = *peekRemoteContext.pLastVersion + 1;
+	request.beginVersion = *peekRemoteContext.pLastVersion;
```

Comment:
do we want to change line 136 instead? i see there are multiple places calling this method, not sure if it has side effects that we do not want

Reply:
`FakeTLog.actor.cpp` might also need some modification, e.g. line 203, line 233 since for it, the last version is inclusive. From `ptxn/TLogServer.actor.cpp`, I also read

```
					lastVersion = std::max(currentVersion, lastVersion);
```

(I can't give an exact line number since I have a bunch of debugging code/modifications at `TLogServer.actor.cpp`, but this is the only line modifying `lastVersion`)

so I am suspecting the returning last version is also inclusive, I am worrying if we might read duplicate version/mutationrefs. But I am only reading the code not running any test, so if the corresponding cursor test passes, I think this is great.

Reply:
RE @hfu5 , I feel lastVersion being exclusive is generally easier to read. I think there are other part of the code that also use exclusive.

Hopefully someday, we will be consistent in all the places...